Realm/RLMConfiguration.h

```diff
@@ -0,0 +1,64 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2015 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+#import <Realm/RLMRealm.h>
+
+RLM_ASSUME_NONNULL_BEGIN
+
+/**
+ An `RLMConfiguration` is used to describe the different options used to
+ create an `RLMRealm` instance.
+ */
+@interface RLMConfiguration : NSObject<NSCopying>
```

Comment:
There's a few things here that talk about the "default configuration", but nothing ever says what that is or what it's used for.

Realm needs to be capitalized.